templates/tutorialv2/view/content.html

```diff
@@ -236,140 +238,172 @@ <h2 class="subtitle">
             {% endif %}
-            <li>
-                <a href="#desactiv-beta" class="open-modal ico-after cross blue">
-                    {% trans "Désactiver la bêta" %}
-                </a>
-                <form action="{% url "content:inactive-beta" content.pk content.slug %}" method="post" class="modal modal-flex" id="desactiv-beta">
-                    {% csrf_token %}
-                    <input type="hidden" name="version" value="{% if version %}{{ version }}{% else %}{{ content.sha_draft }}{% endif %}">
-                    <p>
-                        {% blocktrans with content_title=content.title %}
-                        Êtes-vous certain de vouloir <strong>désactiver</strong> la bêta de ce contenu "<em>{{ content_title }}</em>" ?
-                        {% endblocktrans %}
-                    </p>
-                    <button type="submit">
-                        {% trans "Confirmer" %}
-                    </button>
-                </form>
-            </li>
-        {% endif %}
-
-        {% if beta_topic %}
-            <li>
-                <a href="{{ beta_topic.get_absolute_url }}" class="ico-after online blue">
-                    {% trans "Voir le sujet de bêta" %}
-                </a>
-            </li>
         {% endif %}
+        {# END BETA #}
 
+        {# ONLINE VERSION #}
         {% if public_content_object %}
             <li>
                 <a href="{{ public_content_object.get_absolute_url_online }}" class="ico-after online blue">
                     {% trans "Voir la version en ligne" %}
                 </a>
             </li>
         {% endif %}
+        {# END ONLINE VERSION #}
 
-
+        {# HISTORY #}
         <li>
             <a href="{% url "content:history" content.pk content.slug_repository %}" class="ico-after history blue">
                 {% trans "Historique des versions" %}
             </a>
         </li>
-        {% if not content.in_validation %}
-            {% if content.current_version != content.sha_public %}
+        {# END HISTORY #}
+
+        {# VALIDATION OR PUBLICATION (NO VALIDATION BEFORE) #}
+        {% if content.required_validation_before %}
+            {# Validation (require validation before publication) #}
+            {% if not content.in_validation %}
+                {% if content.current_version != content.sha_public %}
+                    <li>
+                        <a href="#ask-validation" class="open-modal ico-after tick green">
+                            {% trans "Demander la validation" %}
+                        </a>
+                    </li>
+                {% endif %}
+                {% else %}
                 <li>
-                    <a href="#ask-validation" class="open-modal ico-after tick green">
-                        {% trans "Demander la validation" %}
-                    </a>
+                    <a href="#cancel-validation" class="open-modal ico-after cross blue">{% trans "Annuler la validation" %}</a>
```

Comment:
c'est vraiment nécessaire?

2016-06-03 14:18 GMT+02:00 A-312 <notifications@github.com>:

> In templates/tutorialv2/view/content.html
> <https://github.com/zestedesavoir/zds-site/pull/3606#discussion_r65697761>
> :
>
> >                  <li>
> > -                    <a href="#ask-validation" class="open-modal ico-after tick green">
> > -                        {% trans "Demander la validation" %}
> > -                    </a>
> > +                    <a href="#cancel-validation" class="open-modal ico-after cross blue">{% trans "Annuler la validation" %}</a>
>
> (Détail) Il manque 2 retours à la ligne pour "Annuler la validation" :
>
> <a>
>    {% trans "Annuler la validation"}
> </a>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zestedesavoir/zds-site/pull/3606/files/409434b7f3f4433371be1df553908236f1cbf13a#r65697761>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABc_xRTH5hJqFRwbEWgBNkQQw92gz1wpks5qIBuugaJpZM4IdSmR>
> .
>